packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -878,7 +878,9 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   /// The color to use when painting the cursor aligned to the text while
   /// rendering the floating cursor.
   ///
-  /// The default is light grey.
+  /// Typically this would be set to [CupertinoColors.inactiveGray].
```

Comment:
I was going to complain that this is only for Cupertino, but it looks like it's [used for Material](https://github.com/flutter/flutter/blob/0c10e1ca54ae74043909059e2ff56bf5dd0c3d23/packages/flutter/lib/src/material/text_field.dart#L1424) too...